apps/cli-gui/src/app/app.component.html

```diff
@@ -1,5 +1,10 @@
 <ng-container *ngIf="core.coreState$ | async as coreState">
   <mat-toolbar color="primary" class="header">
+    <button mat-icon-button (click)="sidenavService.toggle()">
+      <mat-icon>{{
+        (sidenavService.isExpanded$ | async) ? 'menu' : 'menu_open'
```

Comment:
I think this can be a bit more readable if the icon itself was an observable:

// in component.ts file
```
sidebarMenuIcon$ = this.sidenavService.isExpanded$.pipe(map(expanded => `menu${expanded ? '' : '_open'}`))
```

```suggestion
        sidenavMenuIcon$ | async
```

Reply:
I don't like the coupling of the home & home_open I can put it as observable but as `expanded ? 'home' : 'home_open'`, it would be more readable for you?

Reply:
Yeah I personally prefer as little template logic as possible.

Reply:
done